packages/esm-form-entry-app/src/app/form-schema/form-schema.service.ts

```diff
@@ -27,11 +40,26 @@ export class FormSchemaService {
     }).pipe(
       map(({ unCompiledSchema, formMetadataObject }) => {
         const formSchema: any = unCompiledSchema.form;
+
+        if (formSchema?.translations) {
```

Comment:
This needs to be adapted to the new translations-lookup mechanism. Essentially, when loading the form, the form should have a `resources` section with a list of form resources. Those named "<form_name>_translations_<lang>" are the resource we need to fetch (there should be a link in the result). This is similar to how we pick the schema out of the form object, so hopefully it's straight-forward.

Reply:
As per my above comment, this is no longer how we're storing translations (that is, as part of the form schema) so why is this code still necessary?